ASIM/dev/Parser YAML templates/vimProcessEventTemplate.yaml

```diff
@@ -0,0 +1,94 @@
+Parser:
+  Title: Process event ASIM parser for <product name>
+  Version: '<parser version>'
+  LastUpdated: <parser update date>
+Product:
+  Name: <product name>
+Normalization:
+  Schema: ProcessEvent
+  Version: '<current schema version>'
+references:
+- Title: ASIM Process Event Schema
+  Link: https://aka.ms/ASimProcessEventDoc
+- Title: ASIM
+  Link: https:/aka.ms/AboutASIM
+Description:
+  This ASIM parser supports normalizing the <product name> logs to the ASIM process event normalized schema.
+ParserName: <parser function name>
+ParserParams:
+  - Name: starttime
+    Type: datetime
+    Default: datetime(null)
+  - Name: endtime
+    Type: datetime
+    Default: datetime(null)
+  - Name: commandline_has_any
+    Type: dynamic
+    Default: dynamic([])
+  - Name: commandline_has_all
+    Type: dynamic
+    Default: dynamic([])
+  - Name: commandline_has_any_ip_prefix
+    Type: dynamic
+    Default: dynamic([])
+  - Name: actingprocess_has_any
+    Type: dynamic
+    Default: dynamic([])
+  - Name: targetprocess_has_any
+    Type: dynamic
+    Default: dynamic([])
+  - Name: parentprocess_has_any
+    Type: dynamic
+    Default: dynamic([])
+  - Name: targetusername_has
+    Type: string
+    Default: '*'
+  - Name: actorusername_has
+    Type: string
+    Default: '*'
+  - Name: dvcipaddr_has_any_prefix
+    Type: dynamic
+    Default: dynamic([])
+  - Name: dvchostname_has_any
+    Type: dynamic
+    Default: dynamic([])
+  - Name: eventtype
+    Type: string
+    Default: '*'
+  - Name: disabled
+    Type: bool
+    Default: false
+ParserQuery: |
+  let parser = (
+    starttime:datetime                 = datetime(null)
+    , endtime:datetime                 = datetime(null)
+    , srcipaddr_has_any_prefix:dynamic = dynamic([])
+    , commandline_has_any:dynamic      = dynamic([])
+    , commandline_has_all:dynamic      = dynamic([])
+    , actingprocess_has_any:dynamic    = dynamic([])
+    , targetprocess_has_any:dynamic    = dynamic([])
+    , parentprocess_has_any:dynamic    = dynamic([])
+    , targetusername_has:string        = '*'
+    , actorusername_has:string         = '*'
+    , dvcipaddr_has_any_prefix:dynamic = dynamic([])
+    , eventresult:string               = '*'
```

Comment:
we dont have eventresult in process.
Please add:
hashes_has_any
eventtype
dvchostname_has_any

https://github.com/Azure/Azure-Sentinel/blob/master/Parsers/ASimProcessEvent/Parsers/vimProcessEventMicrosoft365D.yaml

Reply:
Added the requested fields and validated these with the documentation.
`hashes_has_any` does not seem to be part of the ProcessEvent schema.
@vakohl please advice

Reply:
@azurekid hashes_has_any filter can be checked against fields like TargetProcessMD5 , TargetProcessSHA1 etc.